### PR TITLE
style: Fix if-exp-instead-of-or-operator (FURB110)

### PR DESCRIPTION
**Pull request by echoix**
Ruff rule: https://docs.astral.sh/ruff/rules/if-exp-instead-of-or-operator/

### CODE REVIEW

File: python/grass/pygrass/vector/basic.py [hunk 1]
```diff
@@ -134,7 +134,7 @@ def contains(self, point):
         """
         return bool(
             libvect.Vect_point_in_box(
-                point.x, point.y, point.z if point.z else 0, self.c_bbox
+                point.x, point.y, point.z or 0, self.c_bbox
```
Comment (ninsbl) [inline python/grass/pygrass/vector/basic.py:137]: Not sure about  this one, as only points.z is tested...
There are more of this cases below...

**Review by ninsbl — APPROVED** [review]
Looks good in principle, but please checkout my one comment...

File: python/grass/pygrass/vector/basic.py (hunk 1 above)
Comment (echoix) [reply python/grass/pygrass/vector/basic.py:137]: That I'm pretty sure its not problematic. This code wasn't unpacking a tuple, it was calling `libvect.Vect_point_in_box()` with 4 arguments. The pattern I see here, is that if the point coordinates doesn't have a 3rd dimension (z), it sets z to 0 in order to call the function that checks if a point is in a box. I'm pretty sure I have seen that pattern in mathematical formulations of spatial algorithms outside of GRASS

File: man/parser_standard_options.py [hunk 2]
```diff
@@ -218,7 +218,7 @@
     )
     args = parser.parse_args()
 
-    cfile = args.text if args.text else urlopen(args.url, proxies=None)
+    cfile = args.text or urlopen(args.url, proxies=None)
```
[the remark this answers is not in the document]
Comment (echoix) [reply man/parser_standard_options.py:221]: Was there before

**Comment by echoix** [comment]
This one causes a bit more conflicts with my commits that were after, so it would be nice to have it done before I start later this evening

**Review by ninsbl — APPROVED** [review]

**Pull request merged**